virttest/utils_misc.py

```diff
@@ -2639,12 +2639,27 @@ def get_free_disk(session, mount):
     return int(free)
 
 
+def get_win_mon_free_mem(session):
+    """
+    Get Performace Monitored Free memory for Windows.
+
+    :param session: shell Object.
+
+    :return string: freespace M-bytes
+    """
+    cmd = 'typeperf "\Memory\Free & Zero Page List Bytes" -sc 1'
+    output = session.cmd_output(cmd)
```

Comment:
Like "free -m" in linux,  "typeperf" is old for windows, it exists since WinXP, mainly used for writing performance counter data to the command window, or to a supported log file format.
   As a windows built-in cmd, "typeperf" always has a returned value.
   So, from my point of view, no need to check its execution status, thanks.